### ISSUE TITLE
accessibility error

### ISSUE THREAD

**Issue by Hobokens** [derhansen/form_crshield#32]
The tool [WAVE ](https://wave.webaim.org/) gives the following error message 
`A form control does not have a corresponding label`
for the hidden input field and rates this as an accessibility error.
Is there any way to avoid this error message?

**Comment by derhansen** [comment]
I am not able to reproduce the problem. The field is an input field of the type `hidden`, which is not considered by screenreaders and not considered an error in the WAVE tool. What I could reproduce was the WAVE tool claiming a missing label for the hidden input `text` field created by ext:form. Please check, if this is the same scenario you have.

**Comment by Hobokens** [comment]
I have the same scenario.
